pywwt/nbextension/static/interactive_figure/index.html

```diff
@@ -0,0 +1,29 @@
+﻿<!DOCTYPE html >
+<html style="height: 100%; padding: 0px; margin: 0px; background-color: #000000;">
+  <head>
+
+    <meta charset="utf-8" http-equiv="X-UA-Compatible" content="chrome=1, IE=edge"/>
+
+    <title></title>
+    <script src="https://WorldWideTelescope.github.io/pywwt/wwtsdk.js"></script>
+    <script src="https://code.jquery.com/jquery-1.8.3.min.js"></script>
+    <script src="scripts/wwt_json_api.js"></script>
+    <script src="scripts/interactive_figure.js"></script>
+
+    <!-- The following is to avoid scrollbars -->
+    <style type="text/css">
+      body {overflow:hidden;}
+    </style>
+
+    <script>
+        $(document).ready(setHmtlSettings);
+        $(window).resize(setHmtlSettings);
+    </script>
+
+  </head>
+
+  <body onload="initialize()" style="margin: 0; padding: 0">
+      <div id="WWTCanvas" style="width: 100; height: 100; border-style: none; border-width: 0px;"></div>
+      <div id="WWTErrorText"></div>
```

Comment:
I think I'd like to use `kebab-case` identifiers for HTML element ids ... that seems to be the modern web style. So, can we do `wwt-canvas` and `wwt-error-text`? (This is relatively important since I think we can expect people to copy/paste this code a lot.)